### ISSUE TITLE
Tab Editor Height Not Persisting.

### ISSUE THREAD

**Issue by JarrodBlanton** [ChrisRM/material-theme-jetbrains#1338]
### Summary
With the installation of the new IntelliJ Community Edition v. 2019.2.1 there seems to have been some sort of compatibility issue that broke several things. The first issue that happened was upon restarting after the update, the Plugin installer was not able to initialize Material Theme. I was forced to uninstall Material Theme and re-download it. 

After re-downloading the plugin, the tab height in the editor shrank to a non-existant size, not displaying any of the text for the files that were open. 

I found a solution by changing the tab editor height from 32 to 10 (shrinking the height increases the size of the tab) inside of the settings/material_theme.xml file. However, I don't believe this should be a good fix.

**Comment by mallowigi** [comment]
TLDR: please download 4.4.5

There have been some troubles with the release of the EAP, because they changed the way tabs work. Therefore the plugin’s version for the IDE’s RC is 4.3 whereas it is 4.4 for the EAP.

For now it is still a bit confusing since I have to manage two versions simultaneously, therefore you would get the 4.3 update even though you are on the EAP and so on.

I’ll do a cleanup soon, removing all extra versions and leaving only the 4.3 for the RC while continuing from 4.4 for the EAP.

**Comment by matheussilva-hotmart** [comment]
@mallowigi Upgrading does not fix the problem for this build:
IntelliJ IDEA 2019.2.1 (Ultimate Edition)
Build #IU-192.6262.58, built on August 20, 2019

**Comment by Demeng7215** [comment]
Experiencing same issue. Just updated a few seconds ago, tried restarting IDE, still nothing.
![image](https://user-images.githubusercontent.com/43016355/63551106-387f8b00-c502-11e9-996c-7ff5d172ac05.png)

Ill be changing tab height for now I guess.

**Comment by mallowigi** [comment]
I’ll check that again tomorrow. Maybe they changed it again...

For the time being you can use the Material Theme UI Lite

**Comment by willjosefi** [comment]
> Experiencing same issue. Just updated a few seconds ago, tried restarting IDE, still nothing.
> ![image](https://user-images.githubusercontent.com/43016355/63551106-387f8b00-c502-11e9-996c-7ff5d172ac05.png)
> 
> Ill be changing tab height for now I guess.

same issue here... 2019.2.1
Theme version: 4.3.5

**Comment by mgraves9803** [comment]
Just in case it helps, same problem here
![image](https://user-images.githubusercontent.com/27639180/63552626-fbf56480-c4ec-11e9-82fe-9c14a4a221c4.png)
Version: 4.3.5

**Comment by Fulminant** [comment]
Have the same problem.

Material Theme: 4.3.5
WebStorm: 2019.2.1

But I found a temporary solution
![image](https://user-images.githubusercontent.com/11386039/63553118-258cab00-c542-11e9-985b-8623648e4827.png)

**Comment by matheussilva-hotmart** [comment]
This bug is expected on version 4.3.5
It's not expected if you upgrade manually to 4.4.5 as reported by @mallowigi

**Comment by mgraves9803** [comment]
Also, @mallowigi , when I try and install 4.4.5 manually, this happens: 
![image](https://user-images.githubusercontent.com/27639180/63554761-6c06e900-c4f3-11e9-94cc-8f9e6b41d0b9.png)

**Comment by chinaerserver** [comment]
The same to me with Idea Version 2019.2.1 And Material Theme 4.3.5

**Comment by lukaskris** [comment]
same problem with me. with new version intellij and material theme, 
<img width="234" alt="image" src="https://user-images.githubusercontent.com/15793303/63568787-c316c780-c5a0-11e9-9b99-ea868912db8a.png">

**Comment by lukaskris** [comment]
fixed in version 4.4.5 my intellij version 2019.2.1,
just reset to default, and re-setup again.

<img width="258" alt="image" src="https://user-images.githubusercontent.com/15793303/63568909-623bbf00-c5a1-11e9-989f-d5e8499ea92c.png">

**Comment by mallowigi** [comment]
It looks like the RC caught up with the EAP. Please download 4.4.6 and tell me if the problem is solved

**Comment by blackst0ne** [comment]
@mallowigi 

It works. Thanks!

**Comment by matheussilva-hotmart** [comment]
@mallowigi Still not working here in 4.4.6
I tried to remove all intellij configuration files and setup again, but it doesn't work.
![Screenshot from 2019-08-23 12-19-25](https://user-images.githubusercontent.com/52500004/63603648-70d5a680-c5a0-11e9-8e89-c4bbb517cba0.png)
IntelliJ IDEA 2019.2.1 (Ultimate Edition)
Build #IU-192.6262.58, built on August 20, 2019

**Comment by mallowigi** [comment]
Then make sure you have the right tab height in the settings

**Comment by matheussilva-hotmart** [comment]
@mallowigi Thank you for your fast response.
I think it's something related to jdk12. My past installation was from the [Arch User Repository](https://aur.archlinux.org/packages/intellij-idea-ultimate-edition/) using jdk12.
I installed intellij again from the toolbox(wich uses jre11) and now it works fine!

**Comment by matheussilva-hotmart** [comment]
Confirmed. Just switching the jre fix the issue.

IntelliJ IDEA 2019.2.1 (Ultimate Edition)
Build #IU-192.6262.58, built on August 20, 2019
Runtime version: 11.0.4+11 amd64

![Screenshot from 2019-08-23 15-12-11](https://user-images.githubusercontent.com/52500004/63614056-83f47080-c5b8-11e9-8aae-cb2a9e897d81.png)


IntelliJ IDEA 2019.2.1 (Ultimate Edition)
Build #IU-192.6262.58, built on August 20, 2019
Runtime version: 12.0.2+10 amd64

![Screenshot from 2019-08-23 15-17-56](https://user-images.githubusercontent.com/52500004/63614401-54923380-c5b9-11e9-8328-3f7bdcafddc8.png)

**Comment by mgraves9803** [comment]
Can confirm it is now working
IntelliJ IDEA 2019.2.1 (Ultimate Edition)
Build #IU-192.6262.58, built on August 20, 2019
Runtime version: 11.0.3+12-b304.39 amd64
VM: OpenJDK 64-Bit Server VM by JetBrains s.r.o
Windows 10 10.0
